configuration.deploy.yaml

```diff
@@ -8,6 +8,7 @@ plugins: []
 session_parameters:
     ignore_change_ip: false
     secret_key: 96f4628a2e4d0ce26b6352713f8ac8359b838f4800f1972790622899664bd0f4
+    jwt_key: p3JcQcTwcnKxr39t7p6B7cbaE8CvuPPoOj3kE4HyoCrlKM10BLly72X3Q90uJ8kaDb
```

Comment:
The `session_parameters` dictionary is used for Flask sessions configuration. I recommend to put this aside.

This key should also be placed with the appropirate comment in `configuration.example.yaml`